inductiva/utils/data.py

```diff
@@ -229,6 +229,7 @@ def download_file(
     response: urllib3.response.HTTPResponse,
     output_path: pathlib.Path,
     chunk_size=1000,
+    download_size=None,
```

Comment:
missing argument definition in docstring

Reply:
Done